src/defaultsettings.cpp

```diff
@@ -451,6 +451,7 @@ void set_default_settings(Settings *settings)
 
 	settings->setDefault("high_precision_fpu", "true");
 	settings->setDefault("enable_console", "false");
+	settings->setDefault("screen_dpi", "72");
```

Comment:
This is in the section for all platforms. Surely the DPI is auto-detected on most platforms?

Reply:
I just raised this line above.

Reply:
Before it only applies on non-Android platforms, now it applies on all platforms?